### ISSUE TITLE
Add Channel Name to DialogPVRChannelsOSD

### ISSUE THREAD

**Issue by Andihawke** [jurialmunkey/skin.arctic.horizon#107]
Hi jurialmunkey,
its just a little adjustment to DialogPVRChannelsOSD.xml.
maybe you can take something over.
- increased the number of items to 8
- added the ChannelName to Item
- Start/Endtime just in 1 label. maybe there is a better solution. (it's just fine for me)
thanks plonster76
[Includes_View_PVR.zip](https://github.com/jurialmunkey/skin.arctic.horizon/files/4439941/Includes_View_PVR.zip)

**Comment by jurialmunkey** [comment]
Hey,

Thanks for this.

I've incorporated some of the changes - I've combined the start/end time into the one label and added the channel name below label.

However, the vertical size change doesn't fit with the design specs of the skin. All items in the skin need to have a vertical height that is a multiple of 40px to fit within the grid. The 90px itemheight to have 8 items doesn't fit that spec. Going down to 80px is too small for the purpose. That's why there is a 120px item height (plus it also make sure that the channel logos are easily visible).
